content/_releases/22/26.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Product Update - v22.26
+description: Details of the product update v22.26 from 30th of June 2022.
+layout: rel
+version: "22.26"
+section: Product Updates
+category: product-updates
+releaseDate: 2022-06-30
+redirect_from:
+  - /releases/22.26.html
+---
+
+## Improvements and Updates
+
+### Node.js Sailor version `2.6.28`
+
+We released a new version (`2.6.28`) of the Node.js Sailor library version. This
```

Comment:
A new version of the Node.JS Sailor library 2.6.28 is implemented. This provides improved handling of errors generated by the Maester service during upload of lightweight messages. 

This would be even better if we specify WHERE this sailor lib is implemented and WHAT the new error handling enables - the current version of the text is not very informative at all...

Reply:
@KMobbs this is the main library of elastic.io. People who develop components to run on elastic.io know it.

Reply:
And library is not implemented, it is release. Implementation is done when people use it in their component code.

Reply:
Used your text with a difference of just `released` bit implemented

Reply:
> @KMobbs this is the main library of elastic.io. People who develop components to run on elastic.io know it.

:)) and that is approximately what % of Users??

Please use the following text:
A new version of the Node.JS Sailor library 2.6.28 is released. For Users who develop their own components using our SDKs this update provides improved handling of any errors generated by the Maester service during upload of lightweight messages.